sklearn/utils/estimator_checks.py

```diff
@@ -1547,6 +1584,7 @@ def check_estimators_data_not_an_array(name, Estimator, X, y):
 
 
 def check_parameters_default_constructible(name, Estimator):
+    # THIS ONE IS STILL ON CLASSES
```

Comment:
Okay, I guess. At some point in the future, I think we will want a way to specify testing parameters for non-default-constructables.